src/data.js

```diff
@@ -51,16 +52,17 @@ Client.prototype.expandedScopes = function(resolver) {
 /** Get JSON representation of client */
 Client.prototype.json = function(resolver) {
   return {
-    clientId:       this.clientId,
-    description:    this.description,
-    expires:        this.expires.toJSON(),
-    created:        this.details.created,
-    lastModified:   this.details.lastModified,
-    lastDateUsed:   this.details.lastDateUsed,
-    lastRotated:    this.details.lastRotated,
-    scopes:         this.scopes,
-    expandedScopes: this.expandedScopes(resolver),
-    disabled:       !!this.disabled
+    clientId:           this.clientId,
+    description:        this.description,
+    expires:            this.expires.toJSON(),
+    created:            this.details.created,
+    lastModified:       this.details.lastModified,
+    lastDateUsed:       this.details.lastDateUsed,
+    lastRotated:        this.details.lastRotated,
+    deleteOnExpiration: this.details.deleteOnExpiration,
```

Comment:
Add `|| false `, as property is optional in entity. (or just upgrade the entity to next version).